mne/viz/evoked.py

```diff
@@ -1198,3 +1199,398 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+            # calculate the CI
+            sem_array = {}
+            for condition in conditions:
+                # this will fail if evokeds do not have the same structure
+                # (e.g. channel count)
+                if ch_type == 'grad' and picks is not 'gfp':
+                    data = np.asarray([
+                        _merge_grad_data(
+                            evoked_.data[picks, :]).mean(0)
+                        for evoked_ in evokeds[condition]])
+                else:
+                    data = np.asarray([evoked_.data[picks, :].mean(0)
+                                       for evoked_ in evokeds[condition]])
+                sem_array[condition] = _ci(data, ci)
+
+        # get the grand mean
+        evokeds = dict((cond, combine_evoked(evokeds[cond], weights='equal'))
+                       for cond in conditions)
+
+        if picks == 'gfp':
+            warn("Confidence Interval not drawn when plotting GFP.")
+    else:
+        ci = False
+    # we not have dicts for data ('evokeds' - grand averaged Evoked's)
+    # and the CI ('sem_array') with cond name labels
+
+    # let's plot!
+    if ax is None:
+        fig, ax = plt.subplots(1, 1)
+        fig.set_size_inches(8, 6)
+    else:
+        fig = ax.figure
+
+    # style the individual condition time series
+    # first, check if input is valid
+    tags = set([tag for cond in conditions for tag in cond.split("/")])
+    msg = ("Can't map between conditions and the provided {0}. Make sure "
+           "you have provided keys in the format of '/'-separated tags, "
+           "and that these correspond to '/'-separated tags for the condition "
+           "names (e.g., conditions like 'Visual/Right', and styles like "
+           "`colors=dict(Visual='red'))`. The offending tag was '{1}'.")
+    for name_, item in zip(("colors", "linestyles"),
+                           (colors, linestyles)):
+        if isinstance(item, dict):
+            for key in item:
+                for tag in key.split("/"):
+                    if tag not in tags:
+                        raise ValueError(msg.format(name_, tag))
+    if isinstance(styles, dict):
+        for style_ in styles:
+            if style_ not in conditions:
+                raise ValueError("Could not map between `styles` and "
+                                 "conditions. Condition " + style_ +
+                                 " was not found in the supplied data.")
+
+    # second, color
+    # check: is color a list?
+    if (colors is not None and not isinstance(colors, string_types) and
+            not isinstance(colors, dict) and len(colors) > 1):
+        colors = dict((condition, color) for condition, color
+                      in zip(conditions, colors))
+
+    if not isinstance(colors, dict):  # default colors
+        colors_ = ['#e41a1c', '#377eb8', '#4daf4a', '#984ea3', '#ff7f00',
+                   '#1b9e77', '#d95f02', '#7570b3', '#e7298a', '#66a61e']
```

Comment:
How are these colors selected? Maybe this should go to ``defaults.py`` to have consistency in the future.